content/posts/getting-started/prepare-site/index.fr.md

```diff
@@ -3,17 +3,17 @@ title: "Préparer Votre Site"
 date: 2023-11-06T21:44:20+02:00
 menu:
   sidebar:
-    name: Prepare votre site
+    name: Préparer votre site
     identifier: getting-started-prepare-site
-    parent: getting-started
+    parent: Démarrer
```

Comment:
You can not translate this identifier. This defines the article structure. This probably causing duplicate item on the left sidebar.

Reply:
If I change "Démarrer" to "getting-started", the left menu entry will not be translated into French. exact ?

Reply:
OK